### PR TITLE
helm: unify image and tag and add traefik flag

### PR DESCRIPTION
**Pull request by diegodelemos**
Addresses #262.

### CODE REVIEW

**Comment by diegodelemos** [comment]
Connected with https://gitlab.cern.ch/reanahub/deployment/merge_requests/3/.

File: reana/cli.py [hunk 1]
```diff
@@ -1586,3 +1591,77 @@ def python_install_eggs():
     for component in python_cluster_components:
         for cmd in ['python setup.py bdist_egg', ]:
             run_command(cmd, component)
+
+
+@cli.command(name='git-common-packages-upgrade')
```
[the remark this answers is not in the document]
Comment (diegodelemos) [reply reana/cli.py]: IMHO you are right.

File: reana/cli.py [hunk 2]
```diff
@@ -1586,3 +1591,77 @@ def python_install_eggs():
...
+        """Update the specified package version in all affected components."""
+        components_to_update = {
+            'reana-commons': COMPONENTS_USING_SHARED_MODULE_COMMONS,
+            'reana-db': COMPONENTS_USING_SHARED_MODULE_DB
+        }
+        for component in components_to_update[package]:
+            update_setup_py_dep_cmd = (
+                f'LN=`cat setup.py | grep -n -e "{package}.*>=" | cut -f1 -d: ` && '
+                f'sed -i.bk "`echo $LN`s/>=.*,</>={new_version},</" setup.py'
+                f' && rm setup.py.bk')
```
[the remark this answers is not in the document]
Comment (diegodelemos) [reply reana/cli.py]:
```console
$ rg "\.bk"             
deployment/deployment.py
369:            f'sed -i.bk "/mode: 256/d" '
371:            f'rm {temp_dir}/eosxd/templates/daemonset.yaml.bk && '

reana/reana/cli.py
1620:                f'sed -i.bk "`echo $LN`s/>=.*,</>={new_version},</" setup.py'
1621:                f' && rm setup.py.bk')
```

[the remark this answers is not in the document]
Comment (mvidalgarcia) [reply reana/cli.py]: Oh ok, I didn't know this `sed` option. 👍

**Pull request merged**